src/model.c

```diff
@@ -2149,14 +2149,6 @@ void load_texture_by_name(ModelNodeProperty* propertyName, s32 romOffset, s32 si
             break;
         }
 
-        // try appending "tif" - this is a common issue with textures ported from Star Rod mods
-        char tifName[32];
-        strcpy(tifName, textureName);
-        strcat(tifName, "tif");
-        if (strcmp(tifName, header->name) == 0) {
-            break;
```

Comment:
Possibly worth keeping this and having it log an error for a migration period

Reply:
Texture filenames ending in tif is now the expected case, so this wouldn't do anything.

Reply:
![image](https://github.com/star-haven/papermario-dx/assets/9429556/95ea3140-a56a-4331-9ff0-1f3a331b5d82)